docs/data/visualize.md

```diff
@@ -47,11 +47,28 @@ With data query configured, you can use a variety of popular third-party tools t
 
 When you sync captured data to Viam, that data is stored in the Viam organization’s MongoDB Atlas Data Federation instance.
 
-Most third-party visualization tools require the _connection string_ to that database server, and the _credentials_ to authenticate to that server in order to visualize your data.
+Most third-party visualization tools require the _connection URI_ (also called the connection string) to that database server, and the _credentials_ to authenticate to that server in order to visualize your data.
```

Comment:
Oooh this section answers some of the questions I had when I first started reviewing this PR. Will users who land on the query page but don't need to visualize need to find this section?

Reply:
Great callout, this sentence also belongs over in `query`, thank you!